scopes/explorer/ui/base-component-card/deprecation-sticker/deprecation-sticker.module.scss

```diff
@@ -0,0 +1,16 @@
+.deprecated {
+  display: none;
+  position: absolute;
+  top: 7px;
+  right: 12px;
+  background-color: #ffc640;
```

Comment:
--base-color + colorPalette.hunger

Reply:
this is overly complex and hard to use. lets talk about it